apps/mobile/src/libs/components/app/app.tsx

```diff
@@ -7,11 +7,24 @@ import { SafeAreaProvider } from "react-native-safe-area-context";
 import { Provider as StoreProvider } from "react-redux";
 
 import { ToastMessage } from "~/libs/components/components";
+import { RootScreenName } from "~/libs/enums/enums";
 import { useCallback } from "~/libs/hooks/hooks";
 import { store } from "~/libs/packages/store/store";
 import { globalStyles } from "~/libs/styles/styles";
 import { Root as RootNavigation } from "~/navigations/navigations";
 
+const linking = {
+	config: {
+		screens: {
+			[RootScreenName.FORGOT_PASSWORD]: "Forgot Password",
+			[RootScreenName.RESET_PASSWORD]: "/reset-password",
+			[RootScreenName.SIGN_IN]: "Sign In",
+			[RootScreenName.SIGN_UP]: "Sign Up",
+		},
+	},
+	prefixes: ["app://bebalance", "http://localhost:8081"],
```

Comment:
Maybe let's move it to .env? @revenets what do you think? I don't like having `localhost:8081` in our code

Reply:
Agree